AuthenticationExample/AuthenticationExample/HomeView.swift

```diff
@@ -25,22 +25,24 @@ struct HomeView: View {
     
     var body: some View {
         if let portal = portal {
-            NavigationStack {
+            NavigationSplitView {
```

Comment:
A split view makes much more sense than a stack since a split view will automatically choose the number of columns to show. This means in visionOS and Mac Catalyst there will be two columns (which is expected) and narrow-size classes like an iPhone will collapse into a single column. This is the same thing we do in the Samples app.